docs/api-docs/channels/channels-extending-existing.md

```diff
@@ -103,85 +122,75 @@ Accept: application/json
 
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)
 
-<div class="HubBlock--callout">

```

Comment:
I suggest keeping the original formatting since it did not have any rendering issues.